### PR TITLE
Header builder

### PR DESCRIPTION
**Pull request by sushmak02**
### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/admin-nav-menu.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import NavMenu from "@components/NavMenu";
+import React from "react";
```
Comment (sushmak02) [inline src/admin-nav-menu.js:2]: Remove this file

File: src/styles.css [hunk 2]
```diff
@@ -152,6 +167,38 @@ input[type="radio"].hfe-radio-field:focus {
 	}
 }
 
+#e-admin-top-bar-root.e-admin-top-bar--active {
+	display: none!important;
```
Comment (sushmak02) [inline src/styles.css:171]: verify this css, if not needful please remove

**Review by sushmak02 — COMMENTED** [review]
@akshayurankar48 Verify comments on this PR.

File: src/styles.css (hunk 2 above)
Comment (akshayurankar48) [reply src/styles.css]: done @sushmak02

File: src/admin-nav-menu.js (hunk 1 above)
Comment (akshayurankar48) [reply src/admin-nav-menu.js]: deleted the file

**Review by akshayurankar48 — APPROVED** [review]

**Review by vijetaR — APPROVED** [review]

**Review by Bhikule19 — APPROVED** [review]

File: src/index.js [hunk 3]
```diff
@@ -1,12 +1,25 @@
...
-		const root = createRoot(rootElement); // Use createRoot() for React 18
-		root.render(<App />);
-	}
+    const rootElement = document.getElementById("hfe-settings-app");
+    if (rootElement) {
+        const root = createRoot(rootElement); // Use createRoot() for React 18
+        root.render(<App />);
+    }
+
+    if ( "yes" === hfe_admin_data.show_view_all || window.location.href === hfeSettingsData.header_footer_builder || window.location.href === hfeSettingsData.header_footer_builder || "yes" === hfeSettingsData.is_hfe_post ) {
```
Comment (sushmak02) [inline src/index.js:14]: Same condition is added twice - window.location.href === hfeSettingsData.header_footer_builder

Comment (akshayurankar48) [reply src/index.js]: updated pr feedback @sushmak02

**Pull request merged**